### PR TITLE
Add AppVeyor configuration

### LINKED ISSUE #220: Build for Windows on CI

### PR DESCRIPTION
**Pull request by ferjm**
Fixes #220

### CODE REVIEW

**Comment by ferjm** [comment]
@jdm @Manishearth  I need someone with admin privileges on this repo to setup AppVeyor here. Thanks

**Comment by ferjm** [comment]
I tested this on my fork https://ci.appveyor.com/project/ferjm/media

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  - set PATH=%PATH%;C:\Users\appveyor\.cargo\bin
+  - set PKG_CONFIG_PATH=%PKG_CONFIG_PATH%;C:\gstreamer\1.0\x86_64\lib\pkgconfig
+  - set LIB=%LIB%;C:\gstreamer\1.0\x86_64\lib
+  - rustup -V
+  - rustc --version
+  - cargo --version
+
+
+build_script:
+  - cargo build --all-features
```
Comment (Manishearth) [inline appveyor.yml]: shouldn't we run the examples? or is spoofing audio output hard?

Comment (ferjm) [reply appveyor.yml]: Yes, that would be convenient, I'll see if it's possible to run them as a follow-up.

**Pull request merged**